fs/rc/webgui/webgui.go

```diff
@@ -55,17 +55,25 @@ func CheckAndDownloadWebGUIRelease(checkUpdate bool, forceUpdate bool, fetchURL
 		return errors.New("Web GUI path exists, but is a file instead of folder. Please check the path " + extractPath)
 	}
 
+	// Get the latest release details
+	WebUIURL, tag, size, err := GetLatestReleaseURL(fetchURL)
+	if err != nil {
+		return errors.Wrap(err, "Error checking for web gui release update, skipping update")
+	}
+	dat, err := ioutil.ReadFile(tagPath)
+
+	// Check if current version is equal to latest release
+	tagsMatch := err == nil && string(dat) == tag
+
+	if !tagsMatch {
+		fs.Infof(nil, "A release (%s) for gui is present at %s. Use --rc-web-gui-update to update", tag, WebUIURL)
```

Comment:
I think this should probably show your version tag and the new version tag so the user can gauge whether it is worth doing

Reply:
On it